### PR TITLE
[12.0][MIG] stock_ picking_show_backorder

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of module to version 12.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on runbot

File: stock_picking_show_backorder/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import test_stock_picking_show_backorder
```
Comment (cubells) [inline stock_picking_show_backorder/tests/__init__.py]: Add a new blank line please

**Review by cubells — CHANGES_REQUESTED** [review]
minor changes. Ready to approve.

**Comment by bjeficent** [comment]
Comments attended, I have fixed the README too. Thanks for the review.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Pull request merged**